Sources/NIOPosix/PendingDatagramWritesManager.swift

```diff
@@ -452,7 +452,8 @@ final class PendingDatagramWritesManager: PendingWritesManager {
         assert(self.isOpen)
         self.state.append(pendingWrite)
 
-        if self.state.bytes > waterMark.high && channelWritabilityFlag.compareAndExchange(expected: true, desired: false) {
+        if self.state.bytes > waterMark.high,
```

Comment:
NIO style is to prefer `&&` unless a conditional binding or `case` statement is used.

Reply:
I changed the style here because comma is more natural with a line break (or we’ll get an extremely long line here).

Shall I just ignore the length and keep them in the same line?

Reply:
We can break a line after `&&`, and stylistically that's totally fine for NIO.